src/components/modules/modificationsObserver.ts

```diff
@@ -28,7 +28,7 @@ export default class ModificationsObserver extends Module {
   /**
    * Allows to temporary disable mutations handling
    */
-  private disabled: boolean;
+  private disabled = false;
```

Comment:
I use strict mode of TypeScript. 
https://github.com/impact-blue/editor.js/pull/21#discussion_r495694650

Changes of this file are not related to https://github.com/codex-team/editor.js/issues/1339, 
but these changes are more safety for TypeScript.